appservice/src/createAppService/createAppService.ts

```diff
@@ -74,8 +73,8 @@ export async function createAppService(
                 promptSteps.push(new AppServicePlanListStep());
                 promptSteps.push(new LocationListStep());
             } else {
-                if (workspace.workspaceFolders && workspace.workspaceFolders.length === 1) {
-                    // can make smart defaults if only one workspace is opened
+                if (wizardContext.newSiteKind === AppKind.app && !advancedCreation) {
```

Comment:
So you're already in a switch case for `AppKind.app` and an else case for `!advancedCreation` so I don't think you need this `if` at all